ghcide/test/exe/Main.hs

```diff
@@ -4230,6 +4231,52 @@ addSigLensesTests =
             ]
         ]
 
+addSigLensesForWhereClausesTests :: TestTree
+addSigLensesForWhereClausesTests = testGroup
+  "add signature for where clauses"
+  [ testSession "Disbled" $ do
```

Comment:
?

Reply:
It expects no lens if we disabled it, I revise this more clearly.